### DISCUSSION TITLE
Does Spiral Linux come with the latest version of KDE Plasma?

### DISCUSSION THREAD

**Discussion by Dynamitika** [SpiralLinux/SpiralLinux-project#203] [Q&A] [answered]
For example does it come with KDE Plasma 5.26.3 or do I somehow install it?
Does it update automatically to the lastest KDE version?

**Comment by geckolinux** [comment, marked as answer]
SpiralLinux uses unmodified Debian packages. You can search here for the Debian Stable package versions that come with the SpiralLinux ISO:
https://www.debian.org/distrib/packages#search_packages
And/or you can see the versions that you would get by [upgrading the installed system](https://github.com/SpiralLinux/SpiralLinux-project/wiki#switching-from-debian-stable-to-the-testing-or-unstable-branch) to Debian Testing or Debian Unstable.
